flink-cdc-cli/src/main/java/com/ververica/cdc/cli/CliFrontend.java

```diff
@@ -105,25 +104,34 @@ static CliExecutor createExecutor(CommandLine commandLine) throws Exception {
                 additionalJars);
     }
 
-    private static Path getFlinkHome(CommandLine commandLine) {
+    private static Configuration loadFlinkConfiguration(CommandLine commandLine) throws Exception {
+        Optional<Path> flinkHome = getFlinkHome(commandLine);
+        if (flinkHome.isPresent()) {
+            return FlinkEnvironmentUtils.loadFlinkConfiguration(flinkHome.get());
+        }
+        return new Configuration();
```

Comment:
We use Configuration from Flink conf to open checkpoint and submit job to rest client, if Configuration is empty, job could not be submitted to remote cluster, see https://github.com/ververica/flink-cdc-connectors/blob/33b7c85093a69382d23c309a9b209ddae27a2968/flink-cdc-composer/src/main/java/com/ververica/cdc/composer/flink/FlinkPipelineComposer.java#L60.